src/containers/MyNdla/Arena/components/ArenaSingleFlagPage.tsx

```diff
@@ -73,7 +73,8 @@ const ArenaSingleFlagPage = () => {
           {t("myNdla.arena.admin.flags.flaggedPost")}
         </Heading>
         <Text element="p" margin="small">
-          {t("myNdla.arena.admin.flags.inThread")} <SafeLink to={toArenaTopic(topic.id)}>{`"${topic.title}"`}</SafeLink>
+          {t("myNdla.arena.admin.flags.inThread")}{" "}
+          <SafeLink to={routes.myNdla.arenaTopic(topic.id)}>{`"${topic.title}"`}</SafeLink>
```

Comment:
Er dette en konsekvent endring? Eller er det bare Github-diffen som lurer meg?

Reply:
Det ser ut som om det er mellomrommet før SafeLink som blir endra til {" "} siden linja blir for lang. Kanskje bedre å inline mellomrommet i elementet før.